app/controllers/application_controller.rb

```diff
@@ -10,4 +10,10 @@ class ApplicationController < ActionController::Base
   def record_not_found
     render plain: "404 Not Found", status: 404
   end
+
+  def call_rake(task, options = {})
+    options[:rails_env] ||= Rails.env
+    args = options.map { |n, v| "#{n.to_s.upcase}='#{v}'" }
+    system "rake #{task} #{args.join(' ')} --trace 2>&1 >> #{Rails.root}/log/rake.log &"
+  end
```

Comment:
How about something like:

def call_rake(task_name)
  Rake::Task[task_name].invoke
  logger = Logger.new('legacy_sync.log')
  logger.info "#{Time.now} -- #{name}"
end

Reply:
@laurentqro - see my comment above to @govuklaurence re: system calls